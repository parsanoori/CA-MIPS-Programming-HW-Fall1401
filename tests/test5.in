4
604246375
490996385
406640408
658662395