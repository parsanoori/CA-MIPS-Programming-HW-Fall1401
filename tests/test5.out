406640408
490996385
604246375
658662395